.travis.yml

```diff
@@ -5,5 +5,5 @@ env:
     - DOCKER_IMAGE=fedora:24
     - DOCKER_IMAGE=fedora:25
     - DOCKER_IMAGE=fedora:rawhide
```

Comment:
I especially don't like the necessity to change it in two places. I could actually just remove it from .travis.yml, but that would mean losing the build matrix (separate build jobs for each Fedora release), which I think is nice.
But I agree that it's probably better to keep .travis.yml and Makefile.docker synchronized, so I'll probably change both to contain `fedora:24`, `fedora:latest` and `fedora:rawhide`.